### PR TITLE
add an explanatory blurb on the share data page

### PR DESCRIPTION
**Pull request by dannygoldstein**
[ch1273] @scizen9

This PR adds an explanatory blurb to the share data page to explain what the page is and how it works.

### CODE REVIEW

**Review by dmitryduev — CHANGES_REQUESTED** [review]
Could you please add that this is not "transferring a source" (as on the GROWTH marshal), so the source won't show up in another group as people might expect.

**Comment by dannygoldstein** [comment]
@dmitryduev Can you modify the text as needed (can even be via the github text editor) and merge when you are happy?

**Review by dmitryduev — APPROVED** [review]
👍

File: static/js/components/ShareDataForm.jsx [hunk 1]
```diff
@@ -173,6 +173,16 @@ const ShareDataForm = ({ route }) => {
...
           </Link>
         </Typography>
+        <p>
+          This page allows you to share data for {`${route.id}`} with other
+          users or groups. Select the photometry or spectra you would like to
+          share from the list below, then select the users or groups you would
+          like to share the data with. When you click submit, the access
+          permissions on the data will be updated. This page does <em>not</em>
+          reproduce the &ldquo;transfer source&ldquo; functionality of the
+          GROWTH marshal. Data shared via this page will not cause the source to
```
Comment (stefanv) [inline static/js/components/ShareDataForm.jsx:183]: We would like to keep SkyPortal survey-agnostic, so mentioning the GROWTH marshal should not happen here.

Comment (dmitryduev) [reply static/js/components/ShareDataForm.jsx:183]: Ah, good point @stefanv ! @dannygoldstein could you please remove that sentence?

**Pull request merged**